Biotech/DefInjected/ThingDef/Apparel_Packs.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageData>
+  
+  <!-- EN: bandwidth pack -->
+  <Apparel_PackBandwidth.label>комплект подключений</Apparel_PackBandwidth.label>
+  <!-- EN: A backpack containing bandwidth-expansion computer and antenna. It can increase a mechanitor's maximum total bandwidth. -->
+  <Apparel_PackBandwidth.description>Рюкзак, содержащий в себе компьютер и антенну для увеличения максимальной пропускной способности механитора.</Apparel_PackBandwidth.description>
+  
+  <!-- EN: control pack -->
+  <Apparel_PackControl.label>контрольный комплект</Apparel_PackControl.label>
+  <!-- EN: A backpack containing mech control assist computers. It allows a mechanitor to control one additional control group. -->
+  <Apparel_PackControl.description>Рюкзак, содержащий в себе вспомогательные компьютеры для управления механоидами. Даёт механитору возможность управлять одной дополнительной группой механоидов.</Apparel_PackControl.description>
+  
+  <!-- EN: tox pack -->
+  <Apparel_PackTox.label>токсичный комплект</Apparel_PackTox.label>
+  <!-- EN: A reusable backpack containing canisters of reagents and a mechanism that uses them to generate tox gas. When the wearer activates the pack, it will begin spreading tox gas and continue for several seconds until it runs out of reagents.\n\nOnce used, it must be reloaded with chemfuel before it can be used again.\n\nTox gas burns the lungs and eyes, causing a temporary shortness of breath and reduction in sight. Continued exposure to tox gas results in toxic buildup which can be lethal. -->
+  <Apparel_PackTox.description>Многоразовый рюкзак, содержащий в себе баки с реагентами и механизм для генерации токсичного газа. При активации механизм использует реагенты, распыляя токсичный газ в течение нескольких секунд, пока не закончится содержимое в баках.\n\nДля повторного использования комплект необходимо заправить химическим топливом.\n\nТоксичный газ обжигает лёгкие и глаза, вызывая временную одышку и ухудшение зрения, а его длительное воздействие приводит к отравлению токсинами, что чревато летальным исходом.</Apparel_PackTox.description>
```

Comment:
```suggestion
  <Apparel_PackTox.label>токсичный ранец</Apparel_PackTox.label>
  <!-- EN: A reusable backpack containing canisters of reagents and a mechanism that uses them to generate tox gas. When the wearer activates the pack, it will begin spreading tox gas and continue for several seconds until it runs out of reagents.\n\nOnce used, it must be reloaded with chemfuel before it can be used again.\n\nTox gas burns the lungs and eyes, causing a temporary shortness of breath and reduction in sight. Continued exposure to tox gas results in toxic buildup which can be lethal. -->
  <Apparel_PackTox.description>Ранец с баками для реагентов и газогенератором. При активации распыляет токсичный газ в течение нескольких секунд, пока не опустошатся баки.\n\nДля повторного использования ранец необходимо заправить химическим топливом.\n\nТоксичный газ обжигает лёгкие и глаза, вызывая временную одышку и ухудшение зрения, а его длительное воздействие приводит к отравлению токсинами, что чревато летальным исходом.</Apparel_PackTox.description>
```